src/api/routers/participants/participantsAuditTrail.ts

```diff
@@ -0,0 +1,21 @@
+import { Response } from 'express';
+
+import { UserRoleId } from '../../entities/UserRole';
+import { GetParticipantAuditTrail } from '../../services/auditTrailService';
+import { UserParticipantRequest } from '../../services/participantsService';
+
+export async function getAuditTrail(req: UserParticipantRequest, res: Response) {
```

Comment:
nit: can we call this `handleGetAuditTrail` to match the convention in `participantsRouter.tsx`? My thinking is that methods that have a request and response as parameters should have the `handle` prefix.

Reply:
I changed it.

Reply:
I see you changed it to `handleAuditTrail`. Can this be `handleGetAuditTrail`?